src/Compatibility/Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.targets

```diff
@@ -12,12 +12,13 @@
 
   <Target Name="RunPackageValidation"
           AfterTargets="Pack"
-          Condition="$(IsPackable) == 'true'">
+          Condition="'$(IsPackable)' == 'true'">
```

Comment:
should we add the outputs argument here ?

Reply:
Could you elaborate, I'm not understanding what do you mean?

Reply:
Disregard this.